examples/python/imu_signals/README.md

```diff
@@ -3,6 +3,8 @@ title = "IMU signals"
 tags = ["Plots"]
 description = "Log multi dimensional signals under a single entity."
 thumbnail_dimensions = [480, 480]
+channel = "main"
+build_args = ["--seconds=10"]
```

Comment:
This needs a thumbnail URL to be displayed correctly. Please see with @gavrelina for that.